sklearn/linear_model/_base.py

```diff
@@ -46,6 +47,44 @@
 # intercept oscillation.
 
 
+# FIXME in 0.26: variable 'normalize' should be removed from linear models
+# where now normalize=False. default value of 'normalize' should be changed to
+# False in linear models where now normalize=True and should be deprecated
+# change this function accordingly
+def _deprecate_normalize(normalize, default):
+    if normalize == 'deprecated':
+        _normalize = default
+    else:
+        _normalize = normalize
+
+    if default and normalize == 'deprecated' and \
+       parse_version(__version__) < parse_version('0.26'):
```

Comment:
Sorry if I wasn't clear in my previous comment, but we should not need to rely on the version that the user is using. We only want to write code for the next version which is 0.24.

When the deprecation cycle is over (i.e. in 0.26), then we'll update the code to satisfy our needs.

Right now all we want to do is:

- if the current default is False, deprecate and indicate that the parameter will be removed in 0.26
- if the current default is True, deprecate and indicate that the parameter's default will change in 0.26 and that it will be removed in 0.28.

Reply:
does it look alright to you @NicolasHug ?